src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/lecture-unit-management.component.ts

```diff
@@ -239,4 +261,49 @@ export class LectureUnitManagementComponent implements OnInit, OnDestroy {
                 return undefined;
         }
     }
+    /**
+     * Fetches the ingestion state for each lecture unit asynchronously and updates the lecture unit object.
+     */
+    private updateIngestionStates() {
```

Comment:
This might result in many rest calls to the server. Can we merge the request into one rest call that fetches all states?